includes/class-system-info.php

```
@@ -0,0 +1,297 @@
+		 *     @type string $multisite            'Yes' or 'No'.
+		 * }
+		 */
+		public static function get_wordpress(): array {
+			return array(
+				'version'             => get_bloginfo( 'version' ),
+				'environment_type'    => defined( 'WP_ENVIRONMENT_TYPE' ) ? WP_ENVIRONMENT_TYPE : 'production',
+				'permalink_structure' => get_option( 'permalink_structure' ) ? get_option( 'permalink_structure' ) : __( 'Default', 'performance-optimisation' ),
+				'using_https'         => is_ssl() ? __( 'Yes', 'performance-optimisation' ) : __( 'No', 'performance-optimisation' ),
+				'multisite'           => is_multisite() ? __( 'Yes', 'performance-optimisation' ) : __( 'No', 'performance-optimisation' ),
+			);
+		}
```

PHPMD (2.15.0)

[error] The method get_wordpress is not named in camelCase. (undefined)

(CamelCaseMethodName)